declarations/Flutterwave.json

```
@@ -0,0 +1,9 @@
+{
+    "name": "Flutter wave",
+    "documents": {
+        "Privacy Policy": {
+            "fetch": "https://flutterwave.com/us/terms",
+            "select":"div.page__content"
```

I see that the selected content contains “Updated 10 days ago”. This is classical problem (see for example https://github.com/OpenTermsArchive/contrib-declarations/pull/237), as this would trigger a change each day (tomorrow it will read “Updated 11 days ago” etc).

You should thus either:

- replace that data with the actual date through a [filter](https://docs.opentermsarchive.org/contributing-terms/#filter) if the timestamp is present in the DOM (I believe it isn't);
- `remove` the time marker;
- `select` a more specific part.

In this case, the last option seems the most relevant 🙂 

```suggestion
            "select":".page__content__data"
```

---

I had not caught that. But I have addressed it